src/pages/sign-transaction/components/submit-action.tsx

```diff
@@ -19,16 +22,22 @@ function BaseConfirmButton(props: ButtonProps): JSX.Element {
 }
 
 function SubmitActionSuspense(): JSX.Element {
-  const { handleSubmit } = useFormikContext();
-  const error = useTransactionError();
+  const { handleSubmit, values } = useFormikContext<TransactionFormValues>();
+  const { showHighFeeConfirmation, setShowHighFeeConfirmation } = useDrawers();
   const { isLoading } = useLoading(LoadingKeys.SUBMIT_TRANSACTION);
+  const error = useTransactionError();
 
   const isDisabled = !!error;
 
   return (
     <BaseConfirmButton
       data-testid={TransactionsSelectors.BtnConfirmTransaction}
-      onClick={handleSubmit}
+      onClick={() => {
+        if (values.txFee > HIGH_FEE_AMOUNT_STX) {
```

Comment:
`>=`?

Reply:
I was following the issue that said more than 5, but can change for sure? @markmhx

Reply:
Either good for me